src/components/CourseNavMenu/index.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+import { Tab } from '..';
+import { classes } from '../../utils/misc';
+
+import './stylesheet.scss';
+
+export type CourseNavMenuProps = {
+  items: string[];
+  currentItem: number;
+  onChangeItem: (next: number) => void;
+  style?: React.CSSProperties;
+};
+
+export default function CourseNavMenu({
+  items,
+  currentItem,
+  onChangeItem,
+  style,
+}: CourseNavMenuProps): React.ReactElement {
+  return (
+    <div className="course-nav-menu" style={style}>
```

Comment:
Since `CourseNavMenu` is not a class name anymore, let's change "course-nav-menu" to "CourseNavMenu" just to be consistent with other components :)